src/submission/validation.ts

```diff
@@ -93,12 +93,36 @@ export const usingInvalidProgramId = (
   return [];
 };
 
+export const usingInvalidClinicalProgramId = (
```

Comment:
can we refactor this and the `usingInvalidProgram` to pass the info as param to a private method instead of duplicating the whole function